### ISSUE TITLE
Themes From TinyMCE 4 to 5

### ISSUE THREAD

**Issue by aaronskiba** [portagenetwork/roadmap#383]
Multiple GitHub Actions fail for Dependabot pull request https://github.com/portagenetwork/roadmap/pull/280 with the following error:
`ModuleNotFoundError: Module not found: Error: Can't resolve 'tinymce/themes/modern/theme' in '/home/runner/work/roadmap/roadmap/app/javascript/src/utils'`

However, according to Dependabot, "Merging this pull request will resolve [6 Dependabot alerts](https://github.com/portagenetwork/roadmap/security/dependabot?q=package%3Atinymce+manifest%3Ayarn.lock+has%3Apatch) on tinymce including a moderate severity alert."

The errors are due to theme changes in the v4 to v5 TinyMCE migration:
https://www.tiny.cloud/docs/migration-from-4x/#themes
"Most themes provided with TinyMCE 4 have been removed from TinyMCE 5 and are now combined in a new responsive theme called Silver. The Silver theme is enabled by default and contains a set of configurable UI components that can be used to replace the functionality of the TinyMCE 4 themes: Modern, Inline, and Inlite."

**Comment by aaronskiba** [comment]
This issue may be resolved when the v4.1.0 upstream release is merged. According to upstream CHANGELOG.md: https://github.com/DMPRoadmap/roadmap/blob/main/CHANGELOG.md#upgraded-tinymce-to-v6

```
Upgraded TinyMCE to v6
Upgraded TinyMCE to v6 (v5 EOL is April 20 2023)
Adjusted JS code to conform to new TinyMCE version
Adjusted views to work with the new version
Updated variables.scss file to fix issue with button text/background color contrast
Updated blocks/_tables.scss to fix issue with dropdown menu overlap against table
updated config/initializers/assets.rb to copy over the tinymce skins and bootstrap glyphicons to the public directory so that they are accessible by TinyMCE and Bootstrap code
```

**Comment by aaronskiba** [comment]
- https://github.com/portagenetwork/roadmap/pull/530 and merging upstream releases v4.0.0, v4.0.1, and v4.0.2 resolved this issue.
